message-processors/commodityv3-processor/src/main/java/io/edpn/backend/messageprocessor/commodityv3/infrastructure/persistence/mappers/HistoricStationCommodityMarketDatumEntityMapper.java

```diff
@@ -58,5 +80,5 @@ WHERE id IN (
                   AND (next_timestamp = timestamp + INTERVAL '1 microsecond' OR prev_timestamp = timestamp - INTERVAL '1 microsecond')
             )
             """)
-    int deleteObsoleteForStationAndCommodity(@Param("stationId") UUID stationId, @Param("commodityId") UUID commodityId);
```

Comment:
this was changed in a later PR, what has this been reverted to the older naming?